.travis.yml

```diff
@@ -19,6 +19,13 @@ script:
 after_success:
   - yarn pack
 
+before_deploy:
+  - git config --global user.email ${GITHUB_EMAIL}
+  - git config --global user.name ${GITHUB_USER}
+  - git remote set-url origin "https://${GITHUB_TOKEN}@github.com/YOUR/PROJECT.git" > /dev/null 2>&1
```

Comment:
@maurocolella I assume this is incorrect?

Reply:
Sorry.